src/libraries/System.Runtime.InteropServices/tests/ComDisabled/System/Runtime/InteropServices/Marshal/MarshalComDisabledTests.cs

```diff
@@ -0,0 +1,222 @@
+//using System.Runtime.InteropServices.Tests.Common;
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public partial class MarshalComDisabledTests
+    {
+        // internal static Type? GetTypeFromCLSID(Guid clsid, string? server, bool throwOnError)
+        // public static IntPtr CreateAggregatedObject(IntPtr pOuter, object o)
+        // public static IntPtr CreateAggregatedObject<T>(IntPtr pOuter, T o) where T : notnull
+        // public static int ReleaseComObject(object o)
+        // public static int FinalReleaseComObject(object o)
+        // public static object? GetComObjectData(object obj, object key)
+        // public static bool SetComObjectData(object obj, object key, object? data)
+        // public static object? CreateWrapperOfType(object? o, Type t)
+        // public static TWrapper CreateWrapperOfType<T, TWrapper>(T? o)
+        // public static void GetNativeVariantForObject(object? obj, /* VARIANT * */ IntPtr pDstNativeVariant)
+        // public static void GetNativeVariantForObject<T>(T? obj, IntPtr pDstNativeVariant)
+        // public static object? GetObjectForNativeVariant(/* VARIANT * */ IntPtr pSrcNativeVariant)
+        // public static T? GetObjectForNativeVariant<T>(IntPtr pSrcNativeVariant)
+        // public static object?[] GetObjectsForNativeVariants(/* VARIANT * */ IntPtr aSrcNativeVariant, int cVars)
+        // public static T[] GetObjectsForNativeVariants<T>(IntPtr aSrcNativeVariant, int cVars)
+        // public static object BindToMoniker(string monikerName)
+
+        // public static IntPtr /* IUnknown* */ GetIUnknownForObject(object o)
+        // public static IntPtr /* IDispatch */ GetIDispatchForObject(object o)
+        // public static object GetObjectForIUnknown(IntPtr /* IUnknown* */ pUnk)
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
It looks like every test is `[PlatformSpecific(TestPlatforms.Windows)]`. Might make sense to put it on the class.